.ci/config.yml

```diff
@@ -92,3 +92,6 @@ SUPERSET_OAUTH2_CLIENT_SECRET: StkPwE5GzPuyH4Rp
 SUPERSET_ROW_LEVEL_SECURITY_XAPI_GROUP_KEY: xapi_course_id
 SUPERSET_SECRET_KEY: WlgA6QSQKCUNGD9PAaji0JHY
 SUPERSET_XAPI_DASHBOARD_SLUG: openedx-xapi
+ASPECTS_VERSION: 0.45.0
```

Comment:
Is this version going to be managed somehow?

Reply:
Yes, by the bump-version workflow